Modules/socketmodule.c

```diff
@@ -8187,6 +8187,48 @@ socket_exec(PyObject *m)
 
     ADD_INT_MACRO(m, J1939_FILTER_MAX);
 #endif
+#ifdef HAVE_LINUX_CAN_ISOTP_H
+    ADD_INT_MACRO(m, SOL_CAN_ISOTP);
+
+    /* for socket options affecting the socket (not the global system) */
+    ADD_INT_MACRO(m, CAN_ISOTP_OPTS);
+    ADD_INT_MACRO(m, CAN_ISOTP_RECV_FC);
+
+    /* sockopts to force stmin timer values for protocol regression tests */
```

Comment:
I don't understand what this means, and the comment still does not make sense to me when I cross check with the Linux kernel docs. Can you reword it more clearly?